docs/releases/1.8.txt

```diff
@@ -145,6 +145,9 @@ Minor features
   its value from :meth:`~django.contrib.admin.AdminSite.has_permission`,
   indicates whether the user may access the site.
 
+* Related foreign key fields now feature change and delete buttons that open
+  pop-up windows to easen navigation through object relations.
```

Comment:
Here's my take at it: Foreign key dropdowns now have buttons for changing or deleting related objects using a popup.